### PR TITLE
Add option -U, -d aka psql options when don't want use configuration …

### PR DESCRIPTION
**Pull request by dturon**
# Changes <h1>
New switch options when **-c** isn't set:
* **-U** username
* **-d** database

log to file only when **-l** is set

### CODE REVIEW

File: pg_view.py [hunk 1]
```diff
@@ -3262,14 +3266,16 @@ def main():
...
-        pass
-
     # set basic logging
-    logging.basicConfig(format='%(levelname)s: %(asctime)-15s %(message)s', filename=LOG_FILE_NAME)
+    if options.log_file:
+        LOG_FILE_NAME = options.log_file
+
+        # truncate the former logs
+        with open(LOG_FILE_NAME, 'w'):
+            pass
```
Comment (valgog) [inline pg_view.py]: Why do you truncate the logfile and not append?

Comment (dturon) [inline pg_view.py]: Its behavior from original version, i only moved lines ...

**Comment by hjacobs** [comment]
@dturon thanks for the PR, "-U" is good to have :smile:

File: pg_view.py (hunk 1 above)
Comment (alexeyklyukin) [inline pg_view.py]: No point in keeping the pg_view logs for too long

**Pull request merged**